public/app/features/datasources/state/navModel.ts

```diff
@@ -47,7 +48,7 @@ export function buildNavModel(dataSource: DataSourceSettings, plugin: GenericDat
     });
   }
 
-  if (config.licenseInfo.hasLicense) {
+  if (featureEnabled('dspermissions')) {
```

Comment:
Nit. Maybe move all these "magic" strings to constants and/or a separate file?

Reply:
I thought about that also, right now the way we treat feature toggles is similar eg `config.featureToggles['accesscontrol']` so I didn't go the route of trying to define the list centrally.  We could certainly define them as constants in `licensing.ts` if we think there's a benefit to doing that, though that might be something to think about if we do a refactor to unify with feature toggles.

Reply:
Sure make sense